### PR TITLE
Test(connect-popup): passphrase test

### PR DESCRIPTION
**Pull request by karliatto**
## Description
Makes a little change in connect to [do not release device when device was disconnected](https://github.com/trezor/trezor-suite/pull/8830/commits/b2b3ab88359045de16fc2a8e557d220a71263bfc)

Add passphrase e2e test with popup.

## Related Issue

Related to  https://github.com/trezor/trezor-suite/issues/8775

### CODE REVIEW

**Comment by karliatto** [comment]
All connect tests passed https://gitlab.com/satoshilabs/trezor/trezor-suite/-/pipelines/915973997

Removing commit that runs them in this branch.

File: packages/connect-popup/e2e/tests/passphrase.test.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    await popup.click("button[data-test='@passphrase/hidden/submit-button']");
+    // Accept to see Passphrase.
+    await TrezorUserEnvLink.api.pressYes();
+
+    // Confirm Passphrase is correct.
+    await TrezorUserEnvLink.api.pressNo();
+
+    await popup.waitForSelector("div[data-test='@connect-ui/error']", {
+        state: 'visible',
+        timeout: 40000,
```
Comment (mroz22) [inline packages/connect-popup/e2e/tests/passphrase.test.ts]: maybe timeout shouldn't be that long so  that we validate that the error screen really opens within reasonable time?

File: packages/connect-popup/e2e/tests/passphrase.test.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+test.beforeAll(async () => {
+    await TrezorUserEnvLink.connect();
+});
+
+// popup window reference
+let popup: Page;
+
+// Debug mode does not have to be enable since it is default in connect-explorer
+test('input passphrase in popup and device accepts it', async ({ page }) => {
```
Comment (mroz22) [inline packages/connect-popup/e2e/tests/passphrase.test.ts:15]: what about the case 
1. user opens popup, enters passphrase A, finishes call
2. user reloads 3rd party, meaning inframe is reloaded too but device was not disconnected
3. user opens popup, it should ask passphrase, user enters passphrase B

**Review by mroz22 — COMMENTED** [review]
two small comments. good initiative, passphrase flow should be well-covered definitely

**Review by mroz22 — APPROVED** [review]
extending passphrase mismatch test could be added as part of #8739

File: packages/connect-popup/e2e/tests/passphrase.test.ts (hunk 2 above)
Comment (karliatto) [reply packages/connect-popup/e2e/tests/passphrase.test.ts:15]: I add this case to be followed up with https://github.com/trezor/trezor-suite/issues/8739

File: packages/connect-popup/e2e/tests/passphrase.test.ts (hunk 1 above)
Comment (karliatto) [reply packages/connect-popup/e2e/tests/passphrase.test.ts]: Yes, that makes sense. Changed.

**Pull request merged**